src/workerd/io/observer.h

```diff
@@ -62,6 +62,12 @@ class RequestObserver: public kj::Refcounted {
     return kj::mv(client);
   }
 
+  // Used to record dynamic dispatch workers.
```

Comment:
This comment is a bit vague. Would this be called on the side doing the dispatching? Or the target side?